src/pages/LocationsPage.js

```diff
@@ -239,10 +253,10 @@ class LocationsPage extends Component {
                 <TypeLocationsPaper
                   type={0}
                   onRefresh={() => this.props.fetchLocations(this.locationTypes, 0, null)}
-                  onDelete={(l, idx) => this.onDelete(l, idx, 0)}
+                  onDelete={createRegionLocationRight ? (l, idx) => this.onDelete(l, idx, 0) : null}
```

Comment:
Should createRegion be also used to do the check on delete action? Perhaps we should introduce separate variables for all actions or change name from createRegionLocationRight to manageRegionLocationRight?